config/rbac/leader_election_role.yaml

```diff
@@ -4,6 +4,13 @@ kind: Role
 metadata:
   name: leader-election-role
 rules:
+- apiGroups:
+  - ""
+  resources:
```

Comment:
This is needed for [getConfigMapName](https://github.com/gciavarrini/k4e-operator/blob/e50b2c7214cf978c09a111a7f2cddf30b1f603d1/main.go#L299). I tried to add this rule in `config/rbac/role.yaml` but it didn't work.

However, since Moti suggested eliminating this function and using a constant, this change is no longer necessary.